GalaxyZooTouchTable/GalaxyZooTouchTable/UserConsole.xaml.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Input;
+
+namespace GalaxyZooTouchTable
+{
+    /// <summary>
+    /// Interaction logic for UserConsole.xaml
+    /// </summary>
+    public partial class UserConsole : UserControl
+    {
+        ClassificationPanel Classifier { get; set; }
+
+        public UserConsole()
+        {
+            InitializeComponent();
```

Comment:
Is this WIP stuff? Or am I missing where it's defined

Reply:
This will change a bit, but the code here is pretty simple for now. By default, a xaml item only initializes itself through the constructor when added to a project in WPF.